index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div>
+        <p class="header-microverse">Hello Microverse!</p>
```

Comment:
- As per the requirement, please consider using a `h1` tag instead of `p` tag: 
![image](https://user-images.githubusercontent.com/43172164/196752217-b630a37c-2065-4d50-b5fa-d1667f1e1ae8.png)

Reply:
Thanks for the review, I will make corrections as suggested.